Livecheckables/spice-gtk.rb

```diff
@@ -1,4 +1,4 @@
 class SpiceGtk
   livecheck :url   => "https://www.spice-space.org/download/gtk/",
-            :regex => /spice-gtk-([\d.]+)\.tar\.bz2/
+            :regex => /spice-gtk-([\d.]+)\.tar\.xz/
```

Comment:
```suggestion
            :regex => /href=.*?spice-gtk-(\d+(?:\.\d+)+)\.t/
```

Typically the way livecheckables deal with the various `tar` extensions is to simply use `\.t`in the regex.

Past that, we can improve this regex a bit, while we're at it. With index pages, it's best to only match the file names in the `href` attribute, since the file name in text nodes can be truncated. Additionally, the capture group for the version string is basically matching anything (but doesn't intend to) since the `.` is unescaped, so we may as well replace it with the prevailing regex for matching numeric versions.